admin_manual/configuration_database/linux_database_configuration.rst

```diff
@@ -145,6 +144,36 @@ this:
 
 .. _db-postgresql-label:
 
+Configure MySQL for 4-byte Unicode Support
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+For supporting such features as emoji, you have to both enable 4-byte Unicode support in MySQL (instead of the default 3) as well as in ownCloud. 
+To do so:
+
+1. In ``config/config.sample.php`` set ``mysql.utf8mb4`` to true
```

Comment:
To avoid confusion I would explicitly state this is for new installations only.
Q: is there a migration path available?
Q: what about DB backup and restore - any issues?